dex/lexi/table.go

```diff
@@ -53,13 +74,19 @@ func (t *Table) GetRaw(k KV) (b []byte, err error) {
 		}
 		b = d.v
 		return nil
-	})
+	}
+
+	if opts.txn != nil {
+		err = getTxnFunc(opts.txn)
+	} else {
+		err = t.View(getTxnFunc)
+	}
 	return
```

Comment:
Could end like the other one:
```suggestion
	if opts.txn != nil {
		return getTxnFunc(opts.txn)
	}
        return t.View(getTxnFunc)
```